automated-comments/scala/general/proper_function_name.md

```diff
@@ -0,0 +1,4 @@
+Rename the function to `%{functionName}`.
+The function name should match exactly what is used in the tests.
+
+Rename the function to `%<functionName>s`.
```

Comment:
Isn't this a duplicate of the first line?

Reply:
It is, it was moved (reversed order) and this suggestion was not applied (remove this line).

```suggestion
```